src/components/containers/feature.tsx

```diff
@@ -8,7 +8,7 @@ import { DisabledFeatureWrapper } from "styled/react-components/react-components
 /**
  * Component for features
  */
-const Feature: FC<FeatureProps> = ({ feature, children, title, description, strategy }) => {
```

Comment:
Notes from discussion:
- featurize also the info tooltip
- enable required fields from general dialogs
- need to fix the cors issue in order to test this properly